src/templates/Episode.vue

```diff
@@ -0,0 +1,233 @@
+            :content="$page.episode.sponsorship_details.sponsor_offer_details"
+            :code="$page.episode.sponsorship_details.sponsor_offer_code"
+          />
+        </div>
+      </section>
+
+      <section class="episode__section episode__shownotes">
+        <div class="episode__section-inner">
+          <h2>Shownotes</h2>
+          <div 
+            class="episode__shownotes-content"
+            v-html="$page.episode.shownotes"
+          />
+        </div>
+      </section>
+      <section class="episode__section episode__picks">
+        <div class="episode__section-inner">
+          <h2>Our picks this week</h2>
+          <panelists
+            :panelists="panelistsAndPicks"
+            class="episode__panelists"
+          />
+        </div>
+      </section>
+
+      <section class="episode__section">
+        <div class="episode__section-inner">
+          <h2>Transcript</h2>
+          <div
+            class="episode__transcript" 
+            v-html="compiledTranscript"
+          />
+        </div>
+      </section>   
+    </main>
+  </Layout>
+</template>
+
+<page-query>
+query ($id: ID!) {
+  episode(id: $id) {
+    episode_number
+    episode_title
+    date_published
+    cover_art
+    sharing_link
+    download_link
+    rss_link
+    audio_link
+    sponsorship_details {
+      sponsor_name
+      sponsor_logo
+      sponsor_offer_details
+      sponsor_offer_code
+    }
+    picks {
+      chris_picks
+      ben_picks
+      ari_picks
+      elizabeth_picks
+    }
+    shownotes
+    transcript
+  }
+}
+</page-query>
+
+<script>
+import MediaPlayer from '../components/MediaPlayer.vue';
+import SponsorshipBox from '../components/SponsorshipBox.vue';
+import Panelists from '../components/Panelists.vue';
+import * as marked from 'marked';
+
+export default {
+  name: 'EpisodeDetail',
+  components: {
+    MediaPlayer,
+    SponsorshipBox,
+    Panelists,
+  },
+  mounted() {
+    const { chris_picks, ben_picks, ari_picks, elizabeth_picks } = this.$page.episode.picks;
+  },
+  computed: {
+    panelistsAndPicks() {
+      const { chris_picks, ben_picks, ari_picks, elizabeth_picks } = this.$page.episode.picks;
+      
+      const chris = {
+        name: 'Chris Fritz',
+        img: 'images/chris-bio.jpeg',
+        website: 'https://twitter.com/chrisvfritz',
+        picks: [...chris_picks.split(', ')],
+      };
+
+      const ben = {
+        name: 'Ben Hong',
+        img: 'images/ben-bio.jpeg',
+        website: 'https://twitter.com/bencodezen',
+        picks: [...ben_picks.split(', ')],
+      }
+
+      const ari = {
+        name: 'Ari Clark',
+        img: 'images/ari-bio.jpeg',
+        website: 'https://twitter.com/gloomyLumi',
+        picks: [...ari_picks.split(', ')],
+      }
+
+      const elizabeth = {
+        name: 'Elizabeth Fine',
+        img: 'images/elizabeth-bio.jpeg',
+        website: 'https://twitter.com/elizabethfine4',
+        picks: [...elizabeth_picks.split(', ')],
+      }
```

Comment:
this is pretty verbose, i suggest in a further PR we find some way to just 'set' this data in the CMS so we dont have to hardcode it here.